src/Model/File.php

```diff
@@ -124,4 +134,16 @@ public function createFromPath(string $path, string $fileName = null): Model
 
         return $entity;
     }
+
+    public function cleanupDrafts(): void
+    {
+        $this->assertIsModel();
+
+        $files = (clone $this)
+            ->addCondition('status', self::STATUS_DRAFT)
+            ->addCondition('created_at', '<', (new \DateTime())->sub(new \DateInterval('PT' . $this->cleanupDraftsDelay . 'S')));
+        foreach ($files as $file) {
```

Comment:
for performance reasons, this loop must be put inside transaction!

Reply:
Good point, addressed.